src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -667,9 +667,9 @@ internal static TimeSpan CalculateTimeoutTime(DateTime startTime, DateTime curre
 
         private static Dictionary<string, object> GetRestoreTrackingData(ImplicitRestoreReason restoreReason, int requestCount, bool isBulkRestoreCoordinationEnabled, int projectRestoreInfoSourcesCount, DateTime? bulkRestoreCoordinationCheckStartTime, int projectsReadyCheckCount, List<TimeSpan> projectReadyTimings)
         {
-            var bulkRestoreCoordinationTotalTime = bulkRestoreCoordinationCheckStartTime == default ?
-                "0" :
-                (DateTime.UtcNow - bulkRestoreCoordinationCheckStartTime.Value).TotalSeconds.ToString();
+            double bulkRestoreCoordinationTotalTime = bulkRestoreCoordinationCheckStartTime == default ?
```

Comment:
Is there a test for this field?

Reply:
not really. 
We don't have any solution restore worker tests.